### ISSUE TITLE
Ability to annotate individual backup records within the Hub

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#959]
A Hub user has requested that it be possible to annotate individual backup records:

> I have an image that has 4 incremental backups. I want to put a name for each backup to know the state of the VM on that backup:
>
>e.g: After app installing, update app v 1.0.1

**Comment by l-arnold** [comment]
I agree.  Haven't looked in a while but was an important subject a while back for me also.

Using a different system mostly now but since Tklbam storage has gotten more affordable have value.

Need to also be able to restore older system images in case BAM is from earlier versions alsom



________________________________
